pkgs/servers/waterfall/default.nix

```diff
@@ -0,0 +1,88 @@
+{ fetchgit
+, git
+, lib
+, maven
+, mineflake
+, stdenv
+}:
+
+let
+  version = "510";
+  pname = "Waterfall";
+
+  src = fetchgit {
```

Comment:
Can `fetchFromGitHub` be used here, or is the `.git` folder important?

Reply:
Without .git subfolder it will not add commit info to build